### PR TITLE
Added corejs-typeahead@0.11.1

### PR DESCRIPTION
**Pull request by x-yuri**
The relevant issue: #6386

### CODE REVIEW

**Comment by maruilian11** [comment]
hi, @x-yuri 
i'm here to help you send the PR
please modify the following field to the right format for our project:
`"license": "MIT",`
remove the `bugs`, `contributors` and `main` field
thank you for your contribution!

ps. you can use `git rebase` to modify this commit, so you don't have to open a new PR.
feel free to ask any question :smiley:

**Comment by x-yuri** [comment]
Is it okay now?

File: ajax/libs/corejs-typeahead/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "author": {
+    "name": "Twitter, Inc.",
+    "url": "https://twitter.com/twitteross"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "jquery": ">=1.7"
+  },
+  "version": "0.11.1",
+  "npmName": "corejs-typeahead",
```
Comment (PeterDaveHello) [inline ajax/libs/corejs-typeahead/package.json]: > npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/corejs-typeahead

**Comment by x-yuri** [comment]
Deleted `"npmName": "corejs-typeahead"` from `package.json`. Not sure if that was right thing to do. Initial commit was generated with `cdnjs-importer`, if anything.

**Comment by PeterDaveHello** [comment]
So `npmFileMap` should also be removed if there is no npm package of this lib, and if so, please try git auto-update for it.

**Comment by x-yuri** [comment]
Removed `npmFileMap` and added `autoupdate`.

File: ajax/libs/corejs-typeahead/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "dependencies": {
+    "jquery": ">=1.7"
+  },
+  "version": "0.11.1",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/corejavascript/typeahead.js",
+    "basePath": "dist",
+    "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/corejs-typeahead/package.json]: Wrong indent here.

**Comment by PeterDaveHello** [comment]
There is 9 days delay, would you please also rebase your branch with our latest master branch? Thanks.

**Comment by x-yuri** [comment]
I've done rebase. But it appears, I misinterpreted your words. Somehow I thought you asked me to do rebase so that the PR consisted of only one commit. So I've been amending my first commit all this time. And speaking of which, was it a good thing to do? Are PRs with several commits welcome?

**Comment by PeterDaveHello** [comment]
Sorry to misunderstand you, I meant 'rebase on' here, to rebase your branch for this br on the latest master branch. Seems you did it! :+1:

**Pull request merged**